test/lib/docker-file.test.ts

```diff
@@ -15,9 +15,9 @@ test("Dockerfile not supplied", async (t) => {
 });
 
 test("Dockerfile not found", async (t) => {
-  t.rejects(
+  await t.rejects(
     () => dockerFile.readDockerfileAndAnalyse("missing/Dockerfile"),
-    new Error("ENOENT: no such file or directory, open 'missing/Dockerfile'"),
+    undefined,
```

Comment:
I couldn't figure out what the error message is for windows so I just swapped it to undefined since we only care that it throws, if you think this is wrong maybe worth checking what error is returned on windows 😄  but otherwise I'm happy just to leave it as undefined

Reply:
Having a look! 🙂

Reply:
Actually, we don't run this test on windows any more as decided on separate ones